### DISCUSSION TITLE
Is it possible to trigger element:pointerclick or element:pointerdown events for elements on the stencil in JointJS+ 4.0.0?

### DISCUSSION THREAD

**Discussion by goyaljai1** [clientIO/joint#2837] [Q&A]
### Introduction

I am currently using JointJS+ version 4.0.0 and would like to know if it's possible to trigger the `element:pointerclick` or `element:pointerdown` events for elements displayed on the stencil. Are these events supported for stencil elements in this version, or is this functionality unavailable?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
It's possible to listen to the stencil paper events in `v4`.

```ts
stencil.getPaper().on('element:pointerclick', (stencilElementView) => { /* .. */ }):
```

The problem in `v4` with `pointerdown` and `pointerclick` is that the clone is always created on pointerdown (the dragging starts on pointerdown). This was solved in `v4.1` by introducing the [dragThreshold](https://docs.jointjs.com/learn/release-notes/4.1.0#ui-Stencil-dragThreshold) option.
